frontend/model/contracts/group.js

```diff
@@ -141,6 +153,83 @@ function isActionYoungerThanUser (actionMeta: Object, userProfile: ?Object): boo
   return compareISOTimestamps(actionMeta.createdDate, userProfile.joinedDate) > 0
 }
 
+function updateGroupStreaks ({ state, getters }) {
+  const streaks = vueFetchInitKV(state, 'streaks', initGroupStreaks())
+  const cPeriod = getters.groupSettings.distributionDate
+  const thisPeriodPayments = getters.groupPeriodPayments[cPeriod]
+  const noPaymentsAtAll = !thisPeriodPayments
+
+  if (streaks.lastStreakPeriod === cPeriod) return
+  else {
+    Vue.set(streaks, 'lastStreakPeriod', cPeriod)
+  }
+
+  // IMPORTANT! This code must be kept in sync with updateAdjustedDistribution!
+  // TODO: see if it's possible to DRY this with the code inside of updateAdjustedDistribution
```

Comment:
Could you copy this same comment inside of `updateAdjustedDistribution` (except replace `updateAdjustedDistribution` with `updateGroupStreaks`)